raster/src/main/scala/geotrellis/raster/histogram/FastMapHistogram.scala

```diff
@@ -58,9 +60,9 @@ object FastMapHistogram {
   * The [[FastMapHistogram]] class.  Quickly compute histograms over
   * integer data.
   */
-class FastMapHistogram(_size: Int, _buckets: Array[Int], _used: Int, _total: Int)
+class FastMapHistogram(var size: Int, var buckets: Array[Int], var counts: Array[Long], var used: Int, var total: Long)
```

Comment:
This exposes `size`, `buckets`, `counts` as mutable fields, they should be private.

```scala
scala> class Box(var x: Int)
defined class Box

scala> val b = new Box(3)
b: Box = Box@4a1f53f1

scala> b.x
res0: Int = 3

scala> b.x = 5
b.x: Int = 5

scala> b.x
res1: Int = 5
```